.travis.yml

```diff
@@ -93,12 +96,13 @@ matrix:
       env:
         - LINT=1
         - PYTHONWARNINGS=ignore
-      before_install:
-        - . ./ci/travis/ci.sh init
       install:
+        - . ./ci/travis/ci.sh init
+      before_script:
+        - . ./ci/travis/ci.sh lint
         - . ./ci/travis/ci.sh build
       script:
-        - . ./ci/travis/ci.sh lint
+        - true
```

Comment:
if we are just skipping, we can just remove the script block right?

Reply:
Sadly not :( if you do that then it falls back to the default outermost block. I think I can technically put `script: []` but it's not clear to me why that works, it's kind of confusing.

Reply:
I see. Add a comment so people maintaining this in the future knows?

Reply:
Sure